src/devices/danfoss.ts

```diff
@@ -350,6 +348,127 @@ const definitions: Definition[] = [
                 'danfossMultimasterRole'], options);
         },
     },
+    {
+        zigbeeModel: ['0x0210', '0x8040', '0x8041', '0x8042'],
+        model: 'Icon2 MC or RT',
```

Comment:
```suggestion
        model: 'Icon2',
```

Can we use this here?